jobs/rocoto/ocnpost.sh

```diff
@@ -124,25 +121,25 @@ for fhr in $fhrlst; do
   
 
     #break up ocn netcdf into multiple files:  
-    if [ -f $COMOUT/ocn_2D_$VDATE.$ENSMEM.$IDATE.nc ]; then 
-      echo "File $COMOUT/ocn_2D_$VDATE.$ENSMEM.$IDATE.nc already exists" 
+    if [ -f $COMOUT/ocean/ocn_2D_$VDATE.$ENSMEM.$IDATE.nc ]; then 
+      echo "File $COMOUT/ocean/ocn_2D_$VDATE.$ENSMEM.$IDATE.nc already exists"
     else
-      $ncks -x -v vo,uo,so,temp $COMOUT/ocn$VDATE.$ENSMEM.$IDATE.nc $COMOUT/ocn_2D_$VDATE.$ENSMEM.$IDATE.nc
+      ncks -x -v vo,uo,so,temp $COMOUT/ocean/ocn$VDATE.$ENSMEM.$IDATE.nc $COMOUT/ocean/ocn_2D_$VDATE.$ENSMEM.$IDATE.nc
```

Comment:
can we do a status check here and everywhere we perform this `ncks` business?
```
err=$?
[[ $err-ne 0 ]] && exit $err
```

Reply:
Fixed